app/client-v2/src/views/AboutView.vue

```diff
@@ -1,14 +1,55 @@
 <script setup lang="ts"></script>
 
 <template>
-  <div class="p-5 mb-3 border-round-sm surface-card">About Page</div>
+  <div
+    class="flex flex-column justify-content-between p-5 mb-3 border-round surface-card shadow-2"
+    style="max-width: 900px"
+  >
+    <div>
+      <h1 class="text-2xl font-semibold">About</h1>
+      <p class="text-lg">
+        BeeBOP is being developed by the RESIDE group within MRC GIDA as an online interface to the PopPUNK method
+        developed by John Lees (<a
+          href="https://www.ebi.ac.uk/"
+          target="_blank"
+          rel="noopener noreferrer"
+          class="no-underline text-primary hover:underline"
+          >EMBL-EBI</a
+        >) and Nick Croucher (<a
+          href="https://www.imperial.ac.uk/"
+          target="_blank"
+          rel="noopener noreferrer"
+          class="no-underline text-primary hover:underline"
+          >Imperial College London</a
+        >).<br />
+        This work is part of the Global Pneumococcal Sequencing project, led by Stephen Bentley (<a
+          href="https://www.sanger.ac.uk/"
+          target="_blank"
+          rel="noopener noreferrer"
+          class="no-underline text-primary hover:underline"
+          >Wellcome Sanger Institute</a
+        >) and Lesley McGee (<a
+          href="https://www.cdc.gov/"
+          target="_blank"
+          rel="noopener noreferrer"
+          class="no-underline text-primary hover:underline"
+          >CDC</a
+        >), managed by Stephanie Lo (Wellcome Sanger Institute), and funded by the
+        <a
+          href="https://www.gatesfoundation.org/"
+          target="_blank"
+          rel="noopener noreferrer"
+          class="no-underline text-primary hover:underline"
+          >Bill & Melinda Gates Foundation and Wellcome.</a
```

Comment:
Wellcome should be a separate link presumably!

Reply:
haha yeah i thought it was one word my bad